ibm/mas_devops/roles/suite_db2_setup_for_manage/templates/setupdb.sh.j2

```diff
@@ -44,6 +64,8 @@ db2 update db cfg for {{ db2_dbname }} using STMTHEAP AUTOMATIC DEFERRED
 db2 update db cfg for {{ db2_dbname }} using UTIL_HEAP_SZ 10000 DEFERRED
 db2 update db cfg for {{ db2_dbname }} using DATABASE_MEMORY AUTOMATIC DEFERRED
 db2 update db cfg for {{ db2_dbname }} using AUTO_STMT_STATS OFF DEFERRED
+
+###  the STMT_CONC may need to be set to off for sum customers
```

Comment:
This is good information to have, but please elaborate why & how to achieve this

Reply:
Asked the DBA, added the comments regarding why & how to achieve this.